thoth/adviser/report.py

```diff
@@ -76,10 +79,18 @@ def add_product(self, product: Product) -> bool:
 
     def to_dict(self) -> Dict[str, Any]:
         """Convert pipeline report to a dict representation."""
+        stack_info: List[Dict[str, Any]] = []
+        stack_info_metadata = os.getenv("THOTH_ADVISER_METADATA")
+        if stack_info_metadata:
+            try:
+                stack_info = ((json.loads(stack_info_metadata).get("thoth.adviser") or {}).get("stack_info")) or []
```

Comment:
Here you can use the optional second argument to `dict.get(key, default)` instead of `or`

Reply:
The `or` pattern is more bullet proof to null values:

```python
>>> d = {"thoth.adviser": None}
>>> d.get("thoth.adviser", {}).get("stack")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
I see, the value in the dictionary can also be `None` ++